lib/pages/faq/SearchWidget.dart

```diff
@@ -0,0 +1,50 @@
+import 'SearchBar.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+class SearchWidget extends StatefulWidget {
```

Comment:
Move this widget to the `search_bar.dart` file (see comments above). Since `SearchWidget` and `SearchBar` rely on each other, they fit in the same file.